src/components/Paginations.jsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+
+const PAGES_PER_SHOW = 5; // 페이지 네이션에 표시할 페이지 수
+
+function Paginations({ totalPage, currentPage, handlePageChange }) {
+  const currentGroup = Math.ceil(currentPage / PAGES_PER_SHOW); // 페이지 그룹
+  const startPage = (currentGroup - 1) * PAGES_PER_SHOW + 1; // 첫 페이지
+  const endPage = Math.min(startPage + PAGES_PER_SHOW - 1, totalPage); // 마지막 페이지
+
+  /**
+   * 이전 그룹으로 이동
+   */
+  const handlePrevGroup = () => {
+    if (startPage > 1) {
+      handlePageChange(startPage - 1); // 이전 페이지 그룹 마지막으로 이동
+    }
+  };
+
+  /**
+   * 다음 그룹으로 이동
+   */
+  const handleNextGroup = () => {
+    if (endPage < totalPage) {
+      handlePageChange(endPage + 1); // 다음 페이지 그룹 첫 번째로 이동
+    }
+  };
+
+  return (
+    <div className="pagination">
+      <button
+        className="btn-pagination btn-prev"
+        onClick={handlePrevGroup}
+        disabled={startPage === 1} // 첫 번째 그룹인 경우 이전 버튼 비활성화
+      >
+        <span className="sr-only">이전</span>
+      </button>
+
+      {Array.from(
```

Comment:
지금 방법도 좋지만 아래와 같이 좀 더 간략하게 쓸 수 있을것같아요~

```suggestion
      [...Array(endPage - startPage + 1).keys()].map(...)
```